tests/test_filtering.py

```diff
@@ -62,6 +64,13 @@ class WidgetListView(GenericModelView):
 
         filtering = Filtering(
             color=operator.eq,
```

Comment:
`ColumnFilterField` needs to get the column name somehow. If you already build the object, then there's no way for `Filtering` to inject the column name (e.g. what if you share the same `ColumnFilterField` object for two keywords?).